src/client/java/dev/creesch/model/WebsocketMessageBuilder.java

```diff
@@ -76,6 +131,7 @@ public static WebsocketJsonMessage createHistoricChatMessage(
                 messageJson,
                 JsonObject.class
             ))
+            .isPing(false)
```

Comment:
As discussed in #7 - unless we record an `isPing` flag in the database we can't accurately give this field to the client for scrollback.

Reply:
I was thinking of ways we could, but in the end it all makes it more complex than needed. Let's add a column to the database.